modules/core/src/main/java/org/apache/ignite/internal/processors/query/schema/management/SchemaManager.java

```diff
@@ -413,13 +416,13 @@ private static void validateTypeDescriptor(GridQueryTypeDescriptor type) throws
     }
 
     /**
-     * Handle cache destroy.
+     * Handle cache stop.
      *
      * @param cacheName Cache name.
-     * @param rmvIdx Whether to remove indexes.
+     * @param destroy Destroy cache.
      * @param clearIdx Whether to clear the index.
      */
-    public void onCacheDestroyed(String cacheName, boolean rmvIdx, boolean clearIdx) {
+    public void onCacheStopped(String cacheName, boolean destroy, boolean clearIdx) {
```

Comment:
If we do an additional related refactoring ticket, this and similar renamings might be moved there too.